### DISCUSSION TITLE
Windows cmd length limit workaround

### DISCUSSION THREAD

**Discussion by bioinformagician** [vdemichev/DiaNN#1265] [Q&A] [answered]
Hi, absolute huge fan of DIA-NN and this issue might just be me missing this part in the documentation.

I'm trying to analyse 300+ raw files in a single search. I have a script that automatically constructs the cmd line argument to call diann with x settings on the 300+ raw files. But the command line call becomes too long for windows to handle(much more than 8k characters) as I'm using the --f "file_path.raw" on each file. So is there an option to call --f on a txt file with the raw file paths? I couldn't find it in the documentation, and I would strongly prefer not using the --dir as that would require me to copy or move all of these raw files which are scattered around to the same folder.

**Comment by vdemichev** [comment, marked as answer]
Hi,

Please put the --f commands in a config file and reference it with --cfg.

Best,
Vadim

**Reply by bioinformagician** [reply]
Works like a charm, thank you very much for the swift reply!
